docs/installation/network-options.md

```diff
@@ -97,6 +97,19 @@ cat /etc/cni/net.d/10-calico.conflist
 You should see that IP forwarding is set to true.
 
 
+</TabItem>
+<TabItem value="Cilium" default>
+
+When nuking your k3s cluster, before running `k3s-killall.sh` or `k3s-uninstall.sh`, manually remove cilium_host, cilium_net and cilium_vxlan interfaces:
```

Comment:
Put the consequences of failure up here, before the example command.
```suggestion
When before running `k3s-killall.sh` or `k3s-uninstall.sh`, you must manually remove the`cilium_host`, `cilium_net` and `cilium_vxlan` interfaces. If you fail to do this, you may loose network connectivity to the host when K3s is stopped.
```

Reply:
thanks